optaplanner-examples/src/test/java/org/optaplanner/examples/conferencescheduling/solver/ConferenceSchedulingScoreRulesXlsxTest.java

```diff
@@ -58,6 +59,26 @@
 @RunWith(Parameterized.class)
 public class ConferenceSchedulingScoreRulesXlsxTest {
 
+    @Parameterized.Parameters(name = "{4}")
+    public static Collection testSheetParameters() {
```

Comment:
```suggestion
    public static Collection<Object[]> testSheetParameters() {
```
* warnings: -1
* consistency: +1